tests/unit/test_zowe_core.py

```diff
@@ -598,11 +591,18 @@ def test_profile_loading_with_invalid_schema(self, get_pass_func):
         """
         # Setup - copy profile to fake filesystem created by pyfakefs
         with self.assertRaises(ValidationError):
-            custom_file_path = os.path.join(self.custom_dir, "invalid.zowe.config.json")
-            shutil.copy(self.original_invalid_file_path, custom_file_path)
-            shutil.copy(self.original_invalid_schema_file_path, self.custom_dir)
+            custom_file_path = os.path.join(self.custom_dir, "zowe.config.json")
             os.chdir(self.custom_dir)
-
+            with open(self.original_file_path, "r") as f:
+                original_config = commentjson.load(f)
+            original_config["$schema"] = "invalid.zowe.schema.json"
+            original_config["profiles"]["zosmf"]["properties"]["port"] = "10443"
+            with open(os.path.join(self.custom_dir, "invalid.zowe.config.json"), "w") as f:
+                commentjson.dump(original_config, f)
+            with open(self.original_schema_file_path, "r") as f:
+                original_schema = commentjson.load(f)
+            with open(os.path.join(self.custom_dir, "invalid.zowe.schema.json"), "w") as f:
+                commentjson.dump(original_schema, f)
```

Comment:
I have also made the `invalid.zowe.schema.json`, and made some changes to the config file dynamically :relaxed: